scripts/check.sh

```diff
@@ -150,6 +151,22 @@ if [[ "${ALLOW_DIRTY}" == true && "${COMMIT_METHOD}" == "message" ]]; then
   exit 1
 fi
 
+# When travis clones a repo for building, it uses a shallow clone. After the
+# first commit on a non-master branch, TRAVIS_COMMIT_RANGE is not set, master
+# is not available and we need to compute the START_REVISION from the common
+# ancestor of $TRAVIS_COMMIT and origin/master.
+if [[ "${TEST_ONLY}" == true ]]; then
+  if ! [[ -z $TRAVIS_COMMIT_RANGE ]] ; then
+    START_REVISION="$TRAVIS_COMMIT_RANGE"
+  elif ! [[ -z $TRAVIS_COMMIT ]] ; then
+    if ! git rev-parse origin/master >& /dev/null; then
```

Comment:
This is unrelated to whether or not `TRAVIS_COMMIT` is set, isn't it?

`origin/master` only mattered specifically previously because it was the default for `START_REVISION`.